### PR TITLE
doc: add add golang sdk doc

### PR DESCRIPTION
**Pull request by mrrtree**
- Add doc of golang SDK

### CODE REVIEW

File: docs/src/cn/sdk/go.md [hunk 1]
```diff
@@ -1 +1,98 @@
 # Go
+
+## 介绍
+
+CeresDBClient 是 CeresDB 的高性能 Golang 版客户端。
```
Comment (MachaelLee) [inline docs/src/cn/sdk/go.md:5]: go 里面的结构是否还是`CeresDBClient`?

File: docs/src/cn/sdk/go.md [hunk 2]
```diff
@@ -1 +1,98 @@
...
+注意: CeresDB 当前仅支持预创建的 `public` database , 未来会支持多个 database。
+
+### 管理表
+
+CeresDB 使用 SQL 来管理表格，比如创建表、删除表或者新增列等等，这和你在使用 SQL 管理其他的数据库时没有太大的区别。
+
+CeresDB 是一个 Schema-less 的时序数据引擎，你可以不必创建 schema 就立刻写入数据（CeresDB 会根据你的第一次写入帮你创建一个默认的 schema）。
+当然你也可以自行创建一个 schema 来更精细化的管理的表（比如索引等）。
+
+**创建表格的样例**
```
Comment (MachaelLee) [inline docs/src/cn/sdk/go.md:32]: 表格这个词语有点怪，直接改成表如何？
```suggestion
**创建表的样例**
```

File: docs/src/cn/sdk/go.md (hunk 1 above)
Comment (mrrtree) [reply docs/src/cn/sdk/go.md]: fixed

File: docs/src/cn/sdk/go.md (hunk 2 above)
Comment (mrrtree) [reply docs/src/cn/sdk/go.md]: fixed

File: docs/src/cn/sdk/go.md [hunk 3]
```diff
@@ -1 +1,98 @@
...
+## 如何使用
+
+### 初始化客户端
+
+```go
+	client, err := ceresdb.NewClient(endpoint, types.Direct,
+		ceresdb.WithDefaultDatabase("public"), // Client所使用的database
+	)
+```
+
```
Comment (MachaelLee) [inline docs/src/cn/sdk/go.md:22]: Add introduction for other attributes of client, for example: writeMaxRetries, maxInflightPoints?

Comment (mrrtree) [reply docs/src/cn/sdk/go.md:22]: done

**Review by MachaelLee — APPROVED** [review]
LGTM

**Pull request merged**